docs/src/UsingQAT/TargetingQIR.md

```diff
@@ -0,0 +1,248 @@
+# Targeting QIR
+
+An important aspect is that the default values change with the target
```

Comment:
Default values of what? This sentence feels rather out of context if I am just directly going to this page.

Reply:
Updated.